From: Berk Çakar <berk2238@hotmail.com>
Date: Mon, 26 Jul 2021 21:26:32 +0300
Subject: [PATCH] Redirect issues to Pisi Linux repos

---
 src/MainWindow.vala | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/MainWindow.vala b/src/MainWindow.vala
index 0e78521..a70ff16 100644
--- a/src/MainWindow.vala
+++ b/src/MainWindow.vala
@@ -284,23 +284,23 @@ public class Feedback.MainWindow : Gtk.ApplicationWindow {
     static SystemEntry[] system_entries = {
         SystemEntry () {
             name = _("Applications Menu"),
-            issues_url = "https://github.com/elementary/applications-menu/issues/new/choose"
+            issues_url = "https://github.com/pisilinux/main/issues/new/choose"
         },
         SystemEntry () {
             name = _("Lock or Login Screen"),
-            issues_url = "https://github.com/elementary/greeter/issues/new/choose"
+            issues_url = "https://github.com/pisilinux/main/issues/new/choose"
         },
         SystemEntry () {
             name = _("Look & Feel"),
-            issues_url = "https://github.com/elementary/stylesheet/issues/new/choose"
+            issues_url = "https://github.com/pisilinux/main/issues/new/choose"
         },
         SystemEntry () {
             name = _("Multitasking or Window Management"),
-            issues_url = "https://github.com/elementary/gala/issues/new/choose"
+            issues_url = "https://github.com/pisilinux/main/issues/new/choose"
         },
         SystemEntry () {
             name = _("Notifications"),
-            issues_url = "https://github.com/elementary/notifications/issues/new/choose"
+            issues_url = "https://github.com/pisilinux/main/issues/new/choose"
         }
     };
 
-- 
2.24.3 (Apple Git-128)
